### ISSUE TITLE
Block Explorer API - withdrawals require more information

### ISSUE THREAD

**Issue by edd** [vegaprotocol/vega#7264]
# Feature Overview

**In order to** correctly show a user the status of a withdrawal
**We will** need to know if a limit has been hit or delay applied
**So that** the block explorer doesn't imply a transaction can be actioned before it really can be

## Existing state
https://github.com/vegaprotocol/frontend-monorepo/pull/2366 shows the current status of withdrawals. It's divided in to three segments - all three being full means the withdrawal is complete. What this ticket is about is about step 2. The current view implies that basically as soon as a withdrawal's multisig bundle, it is ready to be submitted. However this isn't true:
- [0003-NP-LIMI-004](https://github.com/vegaprotocol/specs/blob/master/non-protocol-specs/0003-NP-LIMI-limits_aka_training_wheels.md#0003-NP-LIMI-004) - `Validators can, via multisig, stop and recommence processing bridge transactions`
    - For a client, this can't be checked without going to the smart contract (which the explorer doesn't currently do), but the core knows the current status due to bridge events.
- [0003-NP-LIMI-005](https://github.com/vegaprotocol/specs/blob/master/non-protocol-specs/0003-NP-LIMI-limits_aka_training_wheels.md#0003-NP-LIMI-005) - `Withdrawal delay network parameter requires a wait between withdrawals creation & submission if it meets or exceeds a threshold`
    - This _could_ be calculated by fetching the network parameter, comparing the size of the withdrawal, and guessing when the withdrawal will be ready, but this would be better done on the API.
- [0003-NP-LIMI-006](https://github.com/vegaprotocol/specs/blob/master/non-protocol-specs/0003-NP-LIMI-limits_aka_training_wheels.md#0003-NP-LIMI-006) - `A withdrawal that is subject to delay can be cancelled by a validator`
    - There is no way for the explorer to check that this has happened?

# Tasks
wip

# Additional Details
tldr *this is an API change not a TX change*

This is part of a set of Block Explorer API requests that change the intent of the Block Explorer API from 'return precisely what is in the command' to 'return the details a client needs to render the appropriate details for a transaction'. Ideally we can go through a proper API planning exercise for this, but this ticket can be addressed as is.
